scripts/slim-builds.sh

```diff
@@ -19,18 +19,3 @@ cat <<-EOF >> ./.cargo/config
 # so we don't need to keep these around.
 incremental = false
 EOF
-
-cat <<-EOF >> ./Cargo.toml
-[profile.release]
-# See defaults https://doc.rust-lang.org/cargo/reference/profiles.html#release
-opt-level = 3
-debug = false
-debug-assertions = false
-overflow-checks = false
-lto = false
-panic = 'unwind'
-# Disabled, see build.incremental
-# incremental = false
-codegen-units = 1
-rpath = false
```

Comment:
In the comment it was noted these measures were added because we hit disk size limits on CI. Is that not a problem anymore (or, `codegen-units` and `lto` trim enough fat)?

Reply:
Not sure is the disk size limit on CI still an issue or not (@jszwedko ?). I just tested default release profile (lto = false, codegen-units = 16) vs PR profile (lto = true, codegen-units = 1).
default: 2_782_000 kb
PR: 2_869_444 kb

Reply:
We can see :smile: The release builds still happen on Github Actions runners so they may still hit disk limits.

Reply:
Again, I think the only real change here is `lto = true`. We were already building with `codgen-units = 1`.